### PR TITLE
Add missing '$' sign

### PR DESCRIPTION
**Pull request by Vathsan**
## Purpose
$subject

### CODE REVIEW

File: en/micro-integrator/docs/use-cases/tutorials/exposing-several-services-as-a-single-service.md [hunk 1]
```diff
@@ -573,8 +573,8 @@ You can now start updating the API resource with the mediation flow.
                    <li>Select <strong>Expression</strong> from the list.
                     </li>
                    <li>
-                      Enter "$ctx:uri.var.appointment_id".
-                      Note that the "$ctx" method is similar to using the <code>get-property</code> method. This method checks in the message context.
+                      Enter <code>$ctx:uri.var.appointment_id<code>.
```
Comment (nilminiwso2) [inline en/micro-integrator/docs/use-cases/tutorials/exposing-several-services-as-a-single-service.md:576]: I don't think the $ sign renders properly in the doc build when we use the <code> tags.
That is why we have used double quotes.

Comment (Vathsan) [reply en/micro-integrator/docs/use-cases/tutorials/exposing-several-services-as-a-single-service.md:576]: Though we have double quotes, it is not rendering properly in the current doc. Do we have any other alternative?

Comment (samgnaniah) [reply en/micro-integrator/docs/use-cases/tutorials/exposing-several-services-as-a-single-service.md:576]: Have you tried with single backticks? That should work.

Comment (Vathsan) [reply en/micro-integrator/docs/use-cases/tutorials/exposing-several-services-as-a-single-service.md]: Fixed in bfae56b70733d657700734f5eccdb02547feea44

**Pull request merged**